app/controllers/SendYourFileController.scala

```diff
@@ -83,6 +83,8 @@ class SendYourFileController @Inject() (
               } yield Redirect(controllers.routes.FilePendingChecksController.onPageLoad())
             case _ => Future.successful(InternalServerError)
           }
+        case (None, _, _, _) =>
```

Comment:
@THM068  i dont think we will be requiring this change.. as we are doing validation on page load on valid xml data...